CMakeLists.txt

```diff
@@ -130,14 +126,6 @@ if ( BUILD_TESTING )
     stlab::development
     stlab::stlab )
 
-  #
```

Comment:
Did you consider putting this into a separate PR?

Reply:
Done. #480